pom.xml

```diff
@@ -934,6 +935,12 @@
         <artifactId>hsqldb</artifactId>
         <version>${hsqldb.version}</version>
       </dependency>
+      <dependency>
```

Comment:
any reason for introducing this snakeyaml dependency?

Reply:
what is a bit confusing is that snakeyaml should be managed by jackson as a transitive dependency.
We should not declare it. 
It was indeed upgraded to 2.2 in the version 2.16: 
https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.16
so we should not have to do it manually. 

I suspect there is a conflicting version of this dependency somewhere else. It'd be nice to understand where it comes from.
Need to make sure upgrading to 2.2 does not break something else.

Reply:
you may want to use 

```
./mvnw dependency:tree
```

to understand what happens

Reply:
Even the ThirdEyeDebugServer fails locally if i bump up the Jackson version and not add snakeyaml as a dependency. I removed the test scope and then the server runs as expected.